charts/connection-manager/templates/ui-deployment.yaml

```diff
@@ -23,6 +23,10 @@ spec:
           env:
             - name: API_BASE_URL
               value: {{ .Values.api.url | quote }}
+            - name: CHECK_SESSION_URL
+              value: {{ .Values.ui.checkSessionUrl | quote }}
+            - name: LOGIN_URL
+              value: {{ .Values.ui.loginUrl | quote }}
```

Comment:
Can you add some default values for these variables to values.yaml file?
I saw commented out names, but I think its better to have some default values there.

Reply:
They should be empty by default. Otherwise we could break things elsewhere.